Backpack/Rating/Classes/BPKRating.swift

```diff
@@ -43,9 +43,9 @@ public class BPKRating: UIView {
         }
     }
 
-    public var title: String {
+    public var title: String? {
```

Comment:
this just makes the title nullable, but doesn't hide the view if it is - is the spacing still correct everywhere?

Reply:
@frugoman There's also a SwiftUI version we need to take care of :D